src/Resources/app/administration/src/component/adyen-refund/adyen-refund.scss

```diff
@@ -1,3 +1,25 @@
+/*
```

Comment:
Missing a double asterisk instead of a single one, so it is a doc block

Reply:
c5faa1a